.github/CONTRIBUTING.md

```
@@ -61,25 +67,29 @@ This repository is organized as a monorepo containing:
 
 2. **Setup backend**
+
    ```bash
    cd packages/backend
    bun dev:setup  # you can choose local development too in prompts
    ```
 
 3. **Configure environment**
    Copy the Convex URL from `packages/backend/.env.local` to `apps/web/.env`:
+
    ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)